### ISSUE TITLE
Remove 'marked' support in WebView queries.

### ISSUE THREAD

**Issue by jmoody** [calabash/calabash#94]
- [WebView Support: marked behaves differently on webviews](https://github.com/calabash/calabash-ios/wiki/06-WebView-Support#marked-behaves-differently-on-web-views)
- [Cannot query UIWebView by accessibilityIdentifier or accessibilityLabel] [#735](https://github.com/calabash/calabash-ios/issues/735)

Requires 0.18.0 release on the server because this is a breaking change.

**Comment by TobiasRoikjer** [comment]
@jmoody This is done right?

**Comment by jmoody** [comment]
Nope.
